tensorflow/training/docker/2.6/py3/Dockerfile.cpu

```diff
@@ -189,7 +188,8 @@ RUN HOME_DIR=/root \
    && chmod +x /usr/local/bin/testOSSCompliance \
    && chmod +x ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh \
    && ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh ${HOME_DIR} ${PYTHON} \
-   && rm -rf ${HOME_DIR}/oss_compliance*
+   && rm -rf ${HOME_DIR}/oss_compliance* \
+   && /opt/conda/bin/jupyter-kernelspec remove -fy python3
```

Comment:
Please add this as a separate RUN statement after the step that installs sparkmagic.